libminifi/test/unit/ProcessorTests.cpp

```diff
@@ -33,10 +35,18 @@ TEST_CASE("Test Find file", "[getfileCreate2]"){
 
 	TestController testController;
 
-	testController.enableDebug();
+	Configure *config = Configure::getConfigure();
 
-	ProvenanceTestRepository repo;
-	TestFlowController controller(repo);
+	config->set(BaseLogger::nifi_log_appender,"rollingappender");
```

Comment:
sure. we can remove that. For my last test, i found test is failed and have trouble to find the log for the test that just ran.